autotest/gdrivers/wcs.py

```diff
@@ -272,6 +281,269 @@ def old_wcs_5():
+                "-oo OuterExtents -oo BufSizeAdjust=0.5 -oo NoGridAxisSwap",
+                "-oo NoGridAxisSwap -oo SubsetAxisSwap",
+                ],
+            'Projwin' : "-projwin 3200000 6670000 3280000 6620000",
+            'Outsize' : "-outsize $size 0",
+            'Coverage' : [
+                'smartsea:eusm2016-EPSG2393', 'smartsea:eusm2016-EPSG2393',
+                'smartsea:eusm2016-EPSG2393', 'smartsea__eusm2016-EPSG2393'],
+            'Versions' : [100, 110, 111, 201]
+        },
+        'MapServer' : {
+            'URL' : 'http://194.66.252.155/cgi-bin/BGS_EMODnet_bathymetry/ows',
+            'Options' : [
+                "-oo INTERLEAVE=PIXEL -oo OriginAtBoundary -oo BandIdentifier=none",
+                "-oo INTERLEAVE=PIXEL -oo OffsetsPositive -oo NrOffsets=2 -oo NoGridAxisSwap -oo BandIdentifier=none",
+                "-oo INTERLEAVE=PIXEL -oo OffsetsPositive -oo NrOffsets=2 -oo NoGridAxisSwap -oo BandIdentifier=none",
+                "-oo INTERLEAVE=PIXEL -oo OffsetsPositive -oo NrOffsets=2 -oo NoGridAxisSwap -oo BandIdentifier=none",
+                "-oo OriginAtBoundary",
+                ],
+            'Projwin' : "-projwin 10 45 15 35",
+            'Outsize' : "-outsize $size 0",
+            'Coverage' : 'BGS_EMODNET_CentralMed-MCol',
+            'Versions' : [100, 110, 111, 112, 201]
+        },
+        'Rasdaman' : {
+            'URL' : 'http://ows.rasdaman.org/rasdaman/ows',
+            'Options' : "",
+            'Projwin' : "-projwin 10 45 15 35",
+            'Outsize' : "-outsize $size 0",
+            'Coverage' : 'BlueMarbleCov',
+            'Versions' : [201]
+        },
+        'Rasdaman2' : {
+            'URL' : 'http://ows.rasdaman.org/rasdaman/ows',
+            'Options' : '-oo subset=unix("2008-01-05T01:58:30.000Z")',
+            'Projwin' : "-projwin 100000 5400000 150000 5100000",
+            'Outsize' : "-outsize $size 0",
+            'Coverage' : 'test_irr_cube_2',
+            'Versions' : [201],
+            'Dimension' : "unix(\"2008-01-05T01:58:30.000Z\")"
+        },
+        'ArcGIS' : {
+            'URL' : 'http://paikkatieto.ymparisto.fi/arcgis/services/Testit/Velmu_wcs_testi/MapServer/WCSServer',
+            'Options' : [
+                "",
+                "-oo NrOffsets=2",
+                "-oo NrOffsets=2",
+                "-oo NrOffsets=2",
+                "-oo UseScaleFactor"
+                ],
+            'Projwin' : "-projwin 181000 7005000 200000 6980000",
+            'Outsize' : "-outsize $size 0",
+            'Coverage' : [2, 2, 2, 2, 'Coverage2'],
+            'Versions' : [100, 110, 111, 112, 201]
+        }
+    }
+
+###############################################################################
+
+def wcs_6():
+    driver = gdal.GetDriverByName('WCS')
+    if driver is None:
+	    return 'skip'
+    print("Generating various URLs from the driver and comparing them to ones")
```

Comment:
We generally avoid tests to emit messages. Perhaps just turn that into a comment